drive/include/drive_debug.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <libhal/serial.hpp>
+#include "ackermann_steering.hpp"
+
+namespace sjsu::drive {
+    /**

```

Comment:
more detail on what settings is, elaborate on tri wheel configuration, and a brief line to explain purpose of serial.